Sources/ValueType.swift

```diff
@@ -49,14 +49,39 @@ extension Int64: ValueType {
 }
 
 extension Array where Element: ValueType {
-    public static func value(from object: Any) throws -> [Element] {
+    public static func value(from object: Any, discardInvalidObjects: Bool = false) throws -> [Element] {
```

Comment:
I'm still getting the feel for Swift3 naming, but does `discardingInvalidObjects` sound better to you?